### DISCUSSION TITLE
Welcome to STC Discussions!

### DISCUSSION THREAD

**Discussion by tylov** [stclib/STC#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Agg242** [comment]
Hi,
I've been developper for oil terminal automation and wrote various softwares with various languages.
I've decided to do some retro coding for the Amiga platform as a hobby, mainly in C. I got tired of writing my own container library over and over for different projects and platforms so I looked for anything interesting out there and found various template libs for C. I did a few tests, the first one being that the library shall build on my Amiga 500 with VBCC. Then I challenged the lib with various exercises from AdventOfCode and got deeply satisfied by STC.
I hope to integrate STC in my projects, mainly for Amiga 68k and Amiga PPC.
A big thank you for your work.

**Reply by tylov** [reply]
Thanks for the kind words! Cool that you are able to use STC on the Amiga platform. Yes, the motivation for STC was for me to "show" that C is a real high-level language (it's also been a hobby/therapy), and may be just as efficient to use as C++. By now, I prefer C/STC over C++ even though I used C++ for decades (I'm biased for sure). Please share your projects at some point, it would be fun to see how you use STC. Also report issues or suggestions you may have.

**Reply by Agg242** [reply]
I knew that early C++ compilers were strongly based on the C preprocessor capabilities, but I'm totally impressed by what you achieved with it. You succeded in "showing" it to me.
